crypto/hash_test.go

```diff
@@ -1,29 +1,38 @@
 package crypto
 
 import (
+	"bytes"
+	"encoding/hex"
 	"testing"
 )
 
 func TestBlake(t *testing.T) {
-	r1 := Blake256("abcd")
-	if r1 != "9606e52f00c679e548b5155af5026f5af4130d7a15c990a791fff8d652c464f5" {
+	input, _ := hex.DecodeString("abcd")
+	r1 := Blake256(input)
+	a1 := []byte("9606e52f00c679e548b5155af5026f5af4130d7a15c990a791fff8d652c464f5")
```

Comment:
Casting a string typed value to byte array type uses UTF8 encoding. Please use `hex.DecodeString`.